py/farm_ng/core/event_client.py

```diff
@@ -91,7 +93,7 @@ def __init__(self, config: EventServiceConfig) -> None:
         self._logger: logging.Logger = logging.getLogger(f"{self.config.name}/client")
 
         # the number of messages sent
-        self._counts: dict[str, int] = {}
+        self._metrics = EventServiceHealthMetrics()
```

Comment:
Shouldn't this use the singleton for metrics?  This will allow any EventService that has EventClients to publish metrics only measurable by clients.

Reply:
Or is this a singleton due to the base class?

Reply:
it's a singleton